website/docs/engine-flink/datastream.md

```diff
@@ -0,0 +1,386 @@
+---
+title: "Datastream API"
+sidebar_position: 6
+---
+
+<!--
+ Copyright (c) 2025 Alibaba Group Holding Ltd.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Fluss Datastream
+## Overview
+The Fluss Datastream API provides a Flink DataStream source implementation for reading data from Fluss tables. It allows you to seamlessly integrate Fluss tables with Flink's DataStream API, enabling you to process data from Fluss in your Flink applications.
+
+Key features of the Fluss Datastream API include:
```

Comment:
```suggestion
Key features of the Fluss Datastream Connector include:
```

It's not Fluss Datastream API, it's Flink DataStream API. Actually, we are providng a DataStream Connector like [this](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/datastream/kafka/).

Please replace all the `Fluss Datastream API` with `Fluss Datastream Connector` in this page.